src/Microsoft.Health.Dicom.Functions/BlobMigration/CopyDurableFunction.Orchestration.cs

```diff
@@ -33,7 +33,7 @@ public partial class CopyDurableFunction
     {
         // The ID should be a GUID as generated by the trigger, but we'll assert here just to make sure!
         EnsureArg.IsNotNull(context, nameof(context)).ThrowIfInvalidOperationId();
-        CopyCheckpoint input = context.GetInput<CopyCheckpoint>();
+        BlobMigrationCheckpoint input = context.GetInput<BlobMigrationCheckpoint>();
```

Comment:
Just as a reminder: this is only valid because we explicitly do not serialize our JSON input with type information (i.e. `"$type": "foo"`). If we did that, which is the default behavior, this change would not be backwards compatible and would require a new function name